zproject/urls.py

```diff
@@ -220,6 +218,12 @@
         {'PATCH': 'patch_bot_backend',
          'DELETE': 'deactivate_bot_backend'}),
 
+) + patterns('zerver.views.upload',
+    url(r'^user_uploads/(?P<realm_id_str>(\d*|unk))/(?P<filename>.*)', 'rest_dispatch',
+            {'GET': 'api_get_uploaded_file'}),
```

Comment:
yeah, probably makes sense to be a general case.  But I don't want to switch any other API endpoints, including the upload path, to it; this one is unusual in that it is used-facing, not just developer-facing.